src/patternfly/components/Divider/examples/Divider.md

```diff
@@ -33,3 +37,5 @@ The divider renders as an `<hr>` by default. It is possible to make the divider
 | Class | Applied to | Outcome |
 | -- | -- | -- |
 | `.pf-c-divider` | `<hr>`, `<li>`, `<div>` | Defines the divider component. |
+| `.pf-m-vertical` | `.pf-c-divider` | Modifies the divider component from horizontal to vertical. This modifier requires that the parent has an explicit or implicit height value. |
+| `.pf-m-stretch` | `.pf-m-vertical` | Modifies the vertical divider to stretch to full height. |
```

Comment:
Can we make note that this requires the parent have a flex layout, or for the divider to be a flex item?